src/main/java/it/gov/pagopa/payhub/activities/connector/debtposition/client/DebtPositionClient.java

```diff
@@ -25,4 +27,12 @@ public DebtPositionDTO finalizeSyncStatus(String accessToken, Long debtPositionI
     public DebtPositionDTO checkAndUpdateInstallmentExpiration(String accessToken, Long debtPositionId){
         return debtPositionApisHolder.getDebtPositionApi(accessToken).checkAndUpdateInstallmentExpiration(debtPositionId);
     }
+
+    public String installmentSynchronize(String accessToken, String origin, InstallmentSynchronizeDTO installmentSynchronizeDTO, Boolean massive) {
```

Comment:
Use the enum until client invoke, here, you will transcode into String
```suggestion
    public String installmentSynchronize(String accessToken, DebtPositionOrigin origin, InstallmentSynchronizeDTO installmentSynchronizeDTO, Boolean massive) {
```